pkg/kn/commands/service/describe.go

```diff
@@ -294,7 +294,7 @@ func orderByConfigurationGeneration(descs []*revisionDesc) []*revisionDesc {
 
 func completeWithLatestRevisions(client clientservingv1.KnServingClient, service *servingv1.Service, revisionsSeen sets.String, descs []*revisionDesc) ([]*revisionDesc, error) {
 	for _, revisionName := range []string{service.Status.LatestCreatedRevisionName, service.Status.LatestReadyRevisionName} {
-		if revisionsSeen.Has(revisionName) {
+		if revisionsSeen.Has(revisionName) || revisionName == "" {
```

Comment:
I would move the check for `revisionName == ""` to the first position as it doesn't make sense to check for an empty name in the has anyway.

Reply:
Good catch! Addressed.